<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentGui</class>
 <widget class="QWidget" name="AgentGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1176</width>
    <height>637</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1176</width>
    <height>637</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agent</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../app/icons.qrc">
    <normaloff>:/agents</normaloff>:/agents</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="toolTipDuration">
   <number>-1</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 251, 253);
font: 8pt &quot;Verdana&quot;;</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>General Information</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>1091</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fees &amp; Salary</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>1091</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bank Information:</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>1091</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>520</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Check Documentd assciated</string>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/exit</normaloff>:/exit</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="docs_button">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>520</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Check Documentd assciated</string>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Docs</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/document</normaloff>:/document</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>520</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Check Documentd assciated</string>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="icon">
    <iconset resource="../app/icons.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="active_checkbox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold</string>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>401</y>
     <width>1005</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Bank Name:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>bank_name_line_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="bank_name_line_edit">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>Account / IBAN:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>iban_line_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="iban_line_edit">
      <property name="minimumSize">
       <size>
        <width>260</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>260</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>BIC/SWIFT:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>swift_line_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="swift_line_edit">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Address:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>address_line_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="address_line_edit">
      <property name="minimumSize">
       <size>
        <width>291</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>Post Code:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>postcode_line_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="postcode_line_edit">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>City:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>city_line_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="city_line_edit">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>State:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>state_line_edit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="state_line_edit">
      <property name="minimumSize">
       <size>
        <width>241</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>Country:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>bank_country_combobox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="bank_country_combobox">
      <property name="minimumSize">
       <size>
        <width>181</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>181</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Routing:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>routing</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QLineEdit" name="routing">
      <property name="minimumSize">
       <size>
        <width>191</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>191</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>241</y>
     <width>234</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_7">
    <property name="labelAlignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>     Fixed Salary:</string>
      </property>
      <property name="buddy">
       <cstring>fixed_salary_spinbox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="fixed_salary_spinbox">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="suffix">
       <string> - €</string>
      </property>
      <property name="maximum">
       <double>99999.990000000005239</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>  % From Profit:</string>
      </property>
      <property name="buddy">
       <cstring>from_profit_spinbox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="from_profit_spinbox">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="suffix">
       <string> - %</string>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>% From Turnover:</string>
      </property>
      <property name="buddy">
       <cstring>from_turnover_spinbox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="from_turnover_spinbox">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="suffix">
       <string> - %</string>
      </property>
      <property name="decimals">
       <number>2</number>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Fixed Per Piece:</string>
      </property>
      <property name="buddy">
       <cstring>fixed_per_piece_spinbox</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="fixed_per_piece_spinbox">
      <property name="minimumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>121</width>
        <height>20</height>
       </size>
      </property>
      <property name="suffix">
       <string> - €</string>
      </property>
      <property name="maximum">
       <double>99999.990000000005239</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="email_line_edit">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>148</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>148</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>      E-Mail:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="fiscal_name_line_edit">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>122</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>437</x>
     <y>122</y>
     <width>73</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fiscal Name:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="agent_country_combobox">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>148</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>201</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>201</width>
     <height>20</height>
    </size>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="maxVisibleItems">
    <number>35</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_number_line_edit">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>148</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>122</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> Fiscal Nº:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>34</x>
     <y>148</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Nº:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>863</x>
     <y>148</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Country:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="fiscal_number_line_edit">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>122</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>281</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>41</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Agent Code:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="agent_code_line_edit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>41</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>agent_code_line_edit</tabstop>
  <tabstop>active_checkbox</tabstop>
  <tabstop>fiscal_number_line_edit</tabstop>
  <tabstop>fiscal_name_line_edit</tabstop>
  <tabstop>phone_number_line_edit</tabstop>
  <tabstop>email_line_edit</tabstop>
  <tabstop>agent_country_combobox</tabstop>
  <tabstop>fixed_salary_spinbox</tabstop>
  <tabstop>from_profit_spinbox</tabstop>
  <tabstop>from_turnover_spinbox</tabstop>
  <tabstop>fixed_per_piece_spinbox</tabstop>
  <tabstop>bank_name_line_edit</tabstop>
  <tabstop>iban_line_edit</tabstop>
  <tabstop>swift_line_edit</tabstop>
  <tabstop>address_line_edit</tabstop>
  <tabstop>postcode_line_edit</tabstop>
  <tabstop>city_line_edit</tabstop>
  <tabstop>state_line_edit</tabstop>
  <tabstop>bank_country_combobox</tabstop>
  <tabstop>routing</tabstop>
  <tabstop>docs_button</tabstop>
  <tabstop>save_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources>
  <include location="../app/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>AgentGui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1055</x>
     <y>662</y>
    </hint>
    <hint type="destinationlabel">
     <x>1123</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
